blog/views.py

```diff
@@ -0,0 +1,44 @@
+from django.views.generic import ListView, DetailView, CreateView, DeleteView, UpdateView
+from django.urls import reverse_lazy, reverse
+from blog.models import Blog
+
+
+class BlogListView(ListView):
+    model = Blog
+    template_name = 'blog/blog_list.html'
+    context_object_name = 'blogs'
+
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        return queryset.filter(publication_attribute="True")
+
+
+class BlogDetailView(DetailView):
+    model = Blog
+
+    def get_object(self, queryset=None):
+        self.object = super().get_object(queryset)
+        self.object.views_quantity +=1
+        self.object.save()
+        return self.object
+
+
+class BlogCreateView(CreateView):
+    model = Blog
+    fields = ("title", "content", "photo", "created_at", "publication_attribute", "views_quantity")
```

Comment:
Поле `created_at` задается автоматически, а `views_quantity` не должен выставлять пользователь - ты его считаешь на бэке
```suggestion
    fields = ("title", "content", "photo", "publication_attribute")
```

Reply:
При обнолвении тоже нужно поправить